terragraph_planner/los/test/test_cylindrical_los_validator.py

```diff
@@ -72,11 +72,24 @@ def test_out_of_distance_range(self) -> None:
             0.0,
         )
 
+    def test_large_el_dev(self) -> None:
```

Comment:
I think it's better to put this test in `test_base_los_validator.py`.

Reply:
I have the test there too. Just seeing the style of the unit tests it seems these checks are repeated in each case.